lib/LaTeXML/Core/Document.pm

```diff
@@ -250,6 +250,7 @@ sub canAutoClose {
     || (($t == XML_ELEMENT_NODE)                              # otherwise must be element
     && !$node->getAttribute('_noautoclose')                   # without _noautoclose
     && ($node->getAttribute('_autoclose')                     # and either with _autoclose
+      || $node->getAttribute('_autoopened')                   # or was autoopened
```

Comment:
Does this mean that the implied "default" now is for any Tag() marked as `autoOpen => 1`, it will also lead to `autoClose => 1` ?

I see we have a special attribute `_noautoclose` that prevents autoclose, but it makes me wonder if we should also respect an explicit `autoClose => 0` from Tag(). Or document that we won't, or...

Reply:
When you put it like that, it does seems too much, although it's probably correct most/all of the time?  The problem element in this case was the `ltx:text` opened for font switches, so better to explicitly mark them as auto-closable, as well.  patched...